.gitmodules

```diff
@@ -1,6 +1,3 @@
-[submodule "submodules/SUSE-Cloud/automation"]
-	path = submodules/SUSE-Cloud/automation
-	url = https://github.com/SUSE-Cloud/automation.git
```

Comment:
I see SES integration still uses this module 
https://github.com/SUSE-Cloud/socok8s/blob/master/playbooks/roles/ses

Reply:
Yeah, we use it for CaaSPv3, so it's still not possoble to remove it